README.md

```diff
@@ -157,7 +159,9 @@ patched_versions:
 * `notes` \[String\] (optional): Internal notes regarding the vulnerability's inclusion in this database.
 
 [CVSSv2]: https://www.first.org/cvss/v2/guide
-[CVSSv3]: https://www.first.org/cvss/user-guide
+[CVSSv3]: https://www.first.org/cvss/v3.0/user-guide
+[CVSSv3]: https://www.first.org/cvss/v3.1/user-guide
```

Comment:
We should probably only link to CVSS v3.1 when mentioning CVSSv3.

Reply:
Add ".0" and ".1" to make it more specific.

Reply:
The text still uses the `[CVSSv3]` link name. That should be changed to `[CVSSv3.1]`. We don't link to `[CVSSv3.0]` anywhere so that link can be removed.

Reply:
fixed